Libraries/NativeAnimation/Nodes/RCTInterpolationAnimatedNode.m

```diff
@@ -15,6 +15,8 @@ @implementation RCTInterpolationAnimatedNode
   __weak RCTValueAnimatedNode *_parentNode;
   NSArray<NSNumber *> *_inputRange;
   NSArray<NSNumber *> *_outputRange;
+  BOOL _clampLeft;
```

Comment:
Can you change this to work with #9366, you can revert the JS changes and assume that you will always get passed `extrapolateLeft` and `extrapolateRight`. Let's also add support for the `identity` value (returns the input) to have full parity with JS.